alfalfa_worker/jobs/modelica/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .create_run import CreateRun
+from .full_run import FullRun
+from .step_run import StepRun
```

Comment:
We can definitely move them out. The only reason they're here is to make the job path shorter. So instead of `alfalfa_worker.jobs.openstudio.create_run.CreateRun` you could just run the job `alfalfa_worker.jobs.openstudio.CreateRun`. Not a big deal either way though